packages/bpk-component-bottom-sheet/src/BpkBottomSheet.module.scss

```diff
@@ -114,8 +112,13 @@ $close-button-width: tokens.bpk-spacing-lg() * 2;
     position: sticky;
     top: 0;
     z-index: tokens.$bpk-zindex-tooltip - 1;
+    padding: tokens.bpk-spacing-sm() 0;
 
-    @include borders.bpk-border-bottom-sm(tokens.$bpk-line-day);
+    @include utils.bpk-themeable-property(
```

Comment:
Avoid seeing scrolling content "through" the padding around the header
![image](https://github.com/Skyscanner/backpack/assets/3943125/427c33a9-a7dc-492d-afcb-0d3b8e74f22d)